code/modules/research/xenobiology/crossbreeding/destabilized.dm

```diff
@@ -0,0 +1,356 @@
+
+/obj/item/slimecross/destabilized/pink/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(!isliving(A) || !iscarbon(thrower))
+		return FALSE //Don't play the noise unless it lands a hit on a eligible mob.
+	var/mob/living/carbon/CThrower = thrower
+	if(iscarbon(A))
+		var/mob/living/carbon/CTarget = A
+		CTarget.help_shake_act(CThrower)
+	else
+		var/mob/living/LTarget = A
+		var/oldIntent = CThrower.a_intent
+		CThrower.a_intent = "help"
+		LTarget.attack_hand(CThrower)
+		CThrower.a_intent = oldIntent
+
+/obj/item/slimecross/destabilized/gold
+	colour = "gold"
+	effect_desc = "Causes goliath-like tendrils to shoot up out of the ground at the tile that was hit."
+
+/obj/item/slimecross/destabilized/gold/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	new /obj/effect/temp_visual/goliath_tentacle(get_turf(A), thrower, FALSE)
+
+/obj/item/slimecross/destabilized/oil
+	colour = "oil"
+	effect_desc = "Causes an IED-style explosion at the impacted tile."
+
+/obj/item/slimecross/destabilized/oil/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	explosion(get_turf(A),-1,-1,2, flame_range = 4)
+
+/obj/item/slimecross/destabilized/black
+	colour = "black"
+	effect_desc = "Temporarily blinds the hit creature, or creates a smoke cloud."
+
+/obj/item/slimecross/destabilized/black/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		var/mob/living/L = A
+		L.blind_eyes(10)
+		to_chat(L, "<span class='warning'>Your vision is suddenly clouded with inky blackness!</span>")
+	else
+		var/datum/effect_system/smoke_spread/smoke = new
+		smoke.set_up(2, get_turf(A))
+		smoke.start()
+
+/obj/item/slimecross/destabilized/lightpink
+	colour = "light pink"
+	effect_desc = "Makes the hit creature briefly passive, or leaves behind a symbol of peace."
+
+/obj/item/slimecross/destabilized/lightpink/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		var/mob/living/L = A
+		if(iscarbon(L))
+			var/mob/living/carbon/C = L
+			C.reagents.add_reagent(/datum/reagent/pax, 5)
+		else
+			var/oldFaction = L.faction
+			L.faction |= thrower.faction
+			addtimer(CALLBACK(src, .proc/depacify, L, oldFaction), 50)
+	else
+		new /obj/effect/decal/cleanable/crayon(get_turf(A), "#[rand_hex_color()]", "peace")
+
+/obj/item/slimecross/destabilized/lightpink/proc/depacify(mob/living/M, var/oldFaction)
+	M.faction = oldFaction
+
+/obj/item/slimecross/destabilized/adamantine
+	colour = "adamantine"
+	effect_desc = "Surrounds the hit person with shards of glass and knocks off their shoes. How evil..."
+
+/obj/item/slimecross/destabilized/adamantine/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	for(var/turf/T in spiral_range(1, get_turf(A)))
+		new /obj/item/shard(T)
+	if(ishuman(A))
+		var/mob/living/carbon/human/H = A
+		var/obj/item/targetShoes = H.get_item_by_slot(SLOT_SHOES)
+		if(targetShoes)
+			H.dropItemToGround(targetShoes)
+
+/obj/item/slimecross/destabilized/rainbow
```

Comment:
Give this an eventual end duration, event if it's something absurd like 20 minutes.